rtlib/src/LandmarkIO.cpp

```diff
@@ -6,6 +6,7 @@ void LandmarkIO::read() {
 }
 
 void LandmarkIO::read_landmarks_file_() {
+    assert(fixedImage_ != NULL && movingImage_ != NULL);
```

Comment:
Throw an exception instead:`if(fixed == nullptr || movingImage == nullptr) throw std::runtime_error("Missing input image")` Or something like that.